.github/workflows/dvc-diff.yml

```diff
@@ -66,35 +73,23 @@ jobs:
         echo -e "### Added images\n" >> report.md
         while IFS= read -r line; do
           echo -e "- $line \n" >> report.md
-          cml-publish --title $line --md "$line" >> report.md < /dev/null
+          echo -e "![](${line/baseline/baseline-new})" >> report.md
```

Comment:
The generated markdown for this is `![](pygmt/tests/baseline-new/test_meca_spec_single_focalmecha.png)` which doesn't show up since it points to a local path. Need to update this somehow?

Reply:
There's this deprecation warning on https://github.com/GenericMappingTools/pygmt/actions/runs/5107768299/jobs/9180964772#step:6:55:

```
Warning: "cml publish" is deprecated since "cml comment" now supports "![inline](./asset.png)"
```

Looks like we should be using `cml comment --publish=true` instead now, xref https://cml.dev/doc/ref/comment?

Reply:
Ah wait, just noticed you did try to use `cml comment` in 0c6bc53f1087395ebd11688f0e9bd95d7b7f0e9a. Nevermind then.